libs/common/types.hpp

```diff
@@ -308,6 +310,11 @@ namespace iroha {
   using privkey_t = blob_t<64>;
 
   struct keypair_t {
+    keypair_t() = default;
```

Comment:
What is the reason to add default constructor?

Reply:
There're issues in the monadic bind usage, e.g [here](https://github.com/hyperledger/iroha/blob/75bc558ef1c9533e46eedd357318e7cc74832be0/libs/crypto/keys_manager_impl.cpp#L68)

Reply:
Thank you, I see. But this stuff looks like a not so good design, isn't it?

Reply:
Yeah, definitely
Anyway I'd better rewrite keys loading directly via `operator>>`, but don't really want to mix it up

Reply:
Sure, I guess, we should talk with @Warchant about the status of crypto and maybe create the task for improving the design of it.